examples/routing_service/routing_service_monitoring/c++11/RoutingServiceMonitoring_subscriber.cxx

```diff
@@ -0,0 +1,388 @@
+/*
+ * (c) Copyright, Real-Time Innovations, 2012.  All rights reserved.
+ * RTI grants Licensee a license to use, modify, compile, and create derivative
+ * works of the software solely for use with RTI Connext DDS. Licensee may
+ * redistribute copies of the software provided that all such copies are subject
+ * to this license. The software is provided "as is", with no warranty of any
+ * type, including any warranty for fitness for any purpose. RTI is under no
+ * obligation to maintain or support the software. RTI shall not be liable for
+ * any incidental or consequential damages arising out of the use or inability
+ * to use the software.
+ */
+
+/* RoutingServiceMonitoring_subscriber.cxx
+
+A subscription example to the Routing Service Monitoring topics
+
+*/
+#include <algorithm>
+#include <iostream>
+#include <map>
+#include <exception>
+
+#include <dds/dds.hpp>
+
+#include "RoutingServiceMonitoring.hpp"
+
+// Get the Configuration sample (rti/service/monitoring/config topic) from its
+// Instance Handle. The reader for that topic is passed as a parameter. Also the
+// Instance Handle.
+RTI::RoutingService::Monitoring::Config getConfig(
+        dds::sub::DataReader<RTI::RoutingService::Monitoring::Config>
+                &configReader,
+        const dds::core::InstanceHandle &handle)
```

Comment:
Reference should be located beside the type
```suggestion
        const dds::core::InstanceHandle& handle)
```
Same for line 31-32 parameter. I could see a few more reference issues below.